README.md

```diff
@@ -1 +1,10 @@
-# icdb-345
\ No newline at end of file
+# icdb-345
+
+STEPS TO RUN
+1.  In your console, cd to ICDB-345
+    Run these commands: 
+        npm i dotenv discord.js
```

Comment:
when you do `npm i dotenv discord.js`, those two package names get added to `package.json` which is committed in the project. so in the future, everyone else only has to do `npm i` and everything from `package.json` will be automatically installed. you only need to use the name if it's the first time the project is using a package.

Reply:
Oh I had no clue. Thanks for the clarification!